src/components/Jobs/JobList/JobList.tsx

```diff
@@ -40,6 +40,19 @@ const JobList = (props: Props) => {
     );
   }
 
+  if (jobSearchList.length === 0 && jobPaginationList.length === 0) {
+    return (
+      <div className="container my-5">
```

Comment:
I think you should create a similar component with <Loading />
Ex: <Empty />

Reply:
Fixed